Kickstarter-iOS/Views/Controllers/CommentRepliesViewControllerTests.swift

```diff
@@ -36,4 +37,56 @@ final class CommentRepliesViewControllerTests: TestCase {
       }
     }
   }
+
+  func testView_CurrentUser_LoggedOut() {
+    combos(Language.allLanguages, [Device.phone4_7inch, Device.pad]).forEach {
+      language, _ in
+      withEnvironment(currentUser: nil, language: language) {
+        let controller = CommentRepliesViewController
+          .configuredWith(comment: .template, project: .template, inputAreaBecomeFirstResponder: false)
+        let (parent, _) = traitControllers(device: .phone4_7inch, orientation: .portrait, child: controller)
```

Comment:
pass in the device from the `forEach` because it's supposed to test both 4.7 inch and iPad.